docs/designs/output.md

```diff
@@ -126,9 +128,9 @@ These are called *browser navigation properties* and are stored in manifest.
 {
     "files": [{
         "url": "/dotnet/api/system.string",
-        "path": "en-us/netstandard-2.0/dotnet/api/system.string.json",
+        "site-path": "en-us/netstandard-2.0/dotnet/api/system.string.json",
         "locale": "en-us",
-        "moniker": "netstandard-2.0",
+        "moniker": "netstandard-2.0", 
 
         // other browser navigation properties...
```

Comment:
How about:
```json
"files": [{
    "source": "system.string.yml",
    "siteUrl": "/dotnet/api/system.string",
    "outputPath": "en-us/netstandard-2.0/dotnet/api/system.string.json",
    "locale": "en-us",
    "moniker": "netstandard-2.0",
    "dependencies": [{
        "type": "link",
        "source": "a.md"
    }]
}]
```

Reply:
it would be a little big,  for difference purpose, i suggest to put them in two sections

Reply:
and dependencies has no relationship with moniker?